src/main/java/org/apache/pulsar/manager/zuul/HttpsClientConfiguration.java

```diff
@@ -46,6 +52,24 @@ public class HttpsClientConfiguration {
 
     @Bean
     public CloseableHttpClient httpClient() throws Exception {
+        LaxRedirectStrategy customLaxRedirectStrategy = new LaxRedirectStrategy() {
+            private final String[] redirectHttpMethods = new String[] {
+                    HttpGet.METHOD_NAME,
+                    HttpPost.METHOD_NAME,
+                    HttpHead.METHOD_NAME,
+                    HttpDelete.METHOD_NAME,
+                    HttpPut.METHOD_NAME
+            };
+            @Override
+            protected boolean isRedirectable(final String method) {
+                for (final String m: redirectHttpMethods) {
```

Comment:
I think we can return `true` directly, don't check the HTTP methods.

Reply:
@nodece Thanks for the review. I have made the suggested changes. Please can you review it again.